app/routes/questionnaire.py

```diff
@@ -211,36 +211,16 @@ def block(schema, questionnaire_store, block_id, list_name=None, list_item_id=No
     except InvalidLocationException:
         return redirect(url_for('.get_questionnaire'))
 
-    if block_handler.rendered_block['type'] == 'RelationshipCollector':
+    if schema.get_block(block_id)['type'] == 'RelationshipCollector':
```

Comment:
Is there a reason this doesn't use `block_handler.block['type']`?

Reply:
No - which one do you prefer?

Reply:
No, there isn't, and your suggestion is easier to read

Reply:
:+1: 

I think it just makes sense to use the block_handler wherever possible since we already have it available.